.ibm/pipelines/value_files/values_showcase.yaml

```
@@ -43,6 +40,24 @@ global:
           disabled: false
         - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-keycloak-backend-dynamic
           disabled: false
+        - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton
+          disabled: false
```

```suggestion
        - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton
          disabled: false
        - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton
          disabled: true
          pluginConfig:
            dynamicPlugins:
              frontend:
                janus-idp.backstage-plugin-tekton:
                  mountPoints:
                    - mountPoint: entity.page.ci/cards
                      importName: TektonCI
                      config:
                        layout:
                          gridColumn: "1 / -1"
                        if:
                          allOf:
                            - isTektonCIAvailable
```

do you need this as well?